fair_mango/dataset/dataset.py

```diff
@@ -159,17 +274,63 @@ def get_data_for_all_groups(self) -> list[dict]:
         return self.groups_data
 
     def get_data_for_one_group(self, sensitive: Sequence[str]) -> pd.DataFrame:
-        """Retrieve data for a specific group
+        """Retrieve data corresponding to a specific demographic group present
+        in the sensitive features.
 
         Parameters
         ----------
         sensitive : Sequence[str]
-            the sensitive group
+            Sequence of sensitive group values corresponding to sensitive
+            features. Ex:
+            - gender -> male, female
+            - race -> white, black, asian...
```

Comment:
I don't find this description of the parameter clear.

You could have something like this (change it so it s clear to you too)

```
Sensitive values must be in the same order as `sensitive_attributes`, and so `sensitive` must be the same length as `sensitive_attributes`. For instance, if your `sensitive_attributes` were `["race", "gender"]`, you can pass `sensitive=["white", "male"]`.
```

Reply:
I ended up changing the variable name from `sensitive` to `sensitive_group` and using the suggested docstring with a few modifications.